MM2RandoLib/Randomizers/Enemies/REnemyWeaknesses.cs

```diff
@@ -61,14 +63,14 @@ public void Randomize(Patch p, Random r)
                 }
             }
 
-            shotP.Shuffle(r);
-            shotH.Shuffle(r);
-            shotA.Shuffle(r);
-            shotW.Shuffle(r);
-            shotB.Shuffle(r);
-            shotQ.Shuffle(r);
-            shotC.Shuffle(r);
-            shotM.Shuffle(r);
+            shotP = in_Seed.Shuffle(shotP).ToList();
```

Comment:
All these new `ToList()` calls makes me wonder if the API could be streamlined a bit. I forgot the signature of `Shuffle()` but maybe it should be returning an `IEnumerable` and the consumer expecting the same?

Reply:
This is pretty standard Linq.  Linq methods return IEnumerables, and then the caller can decide how to use that enumerable, which Linq also provides.  It happens that a lot of times Lists are the convenient way to go, but not all the time.  I think keeping to the standard Linq return value is appropriate.